torch/nn/quantized/modules/conv.py

```diff
@@ -126,14 +129,21 @@ def from_float(cls, mod):
         if hasattr(mod, 'weight_fake_quant'):
             # assert type(mod) == cls.__QAT_MODULE, ' nnq.' + cls.__name__ + '.from_float only works for ' + \
             #     cls.__QAT_MODULE.__name__
-            assert hasattr(mod, 'observer'), 'Input float module must have observer attached'
+            if type(mod) == nniqat.ConvBn2d:
+                mod.weight, mod.bias = \
+                    fuse_conv_bn_weights(mod.weight, mod.bias, mod.running_mean,
+                                         mod.running_var, mod.eps, mod.gamma, mod.beta)
+            assert hasattr(mod, 'observer'), 'Input QAT module must have observer attached'
             weight_observer = mod.weight_fake_quant
         else:
-            assert type(mod) == cls.__FLOAT_MODULE, ' nnq.' + cls.__name__ + '.from_float only works for ' + \
-                cls.__FLOAT_MODULE.__name__
+            assert type(mod) == cls._FLOAT_MODULE, ' nnq.' + cls.__name__ + '.from_float only works for ' + \
+                cls._FLOAT_MODULE.__name__
             assert hasattr(mod, 'qconfig'), 'Input float module must have qconfig defined'
-            assert hasattr(mod, 'observer'), 'Input float module must have observer attached'
             weight_observer = mod.qconfig.weight()
+            # workaround for sequential, ConvReLU2d should probably
+            # inherit from Conv2d instead
+            if type(mod) == nni.ConvReLU2d:
+                mod = mod[0]
```

Comment:
We have a problem when we deal with nn._intrinsic.convReLU2d. We want to use the observer after the ReLU, but here we end up using the observer attached to the conv.